css/fonts.css

```diff
@@ -0,0 +1,43 @@
+@font-face {
```

Comment:
It's good practise to split css by sense. But when we have only one css it's better to move it to styles.css. We will do it in future (when we will start using scss preprocessor)